### PR TITLE
Add cert manager to release bundle

### PR DESCRIPTION
**Pull request by a-cool-train**
*Description of changes:*
- Add cert-manager to release bundle
- Copy cert-manager to prod account

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: generatebundlefile/data/input_123_prod.yaml [hunk 1]
```diff
@@ -16,6 +16,14 @@ packages:
         registry: public.ecr.aws/eks-anywhere
         versions:
             - name: 0.21.1-971c31238aeea5f1bda6ec95caaf24993b304157
+  - org: cert-manager
+    projects:
+      - name: cert-manager
+        workloadonly: true
+        repository: cert-manager/cert-manager
+        registry: public.ecr.aws/l0g8r8j6
```
Comment (vincentni) [inline generatebundlefile/data/input_123_prod.yaml:24]: `public.ecr.aws/eks-anywhere`?

File: generatebundlefile/data/input_122_prod.yaml [hunk 2]
```diff
@@ -16,6 +16,14 @@ packages:
         registry: public.ecr.aws/eks-anywhere
         versions:
             - name: 0.21.1-971c31238aeea5f1bda6ec95caaf24993b304157
+  - org: cert-manager
+    projects:
+      - name: cert-manager
+        workloadonly: true
+        repository: cert-manager/cert-manager
+        registry: public.ecr.aws/l0g8r8j6
```
Comment (vincentni) [inline generatebundlefile/data/input_122_prod.yaml:24]: `public.ecr.aws/eks-anywhere`?

File: generatebundlefile/data/input_120_prod.yaml [hunk 3]
```diff
@@ -16,6 +16,14 @@ packages:
         registry: public.ecr.aws/eks-anywhere
         versions:
             - name: 0.21.1-971c31238aeea5f1bda6ec95caaf24993b304157
+  - org: cert-manager
+    projects:
+      - name: cert-manager
+        workloadonly: true
+        repository: cert-manager/cert-manager
+        registry: public.ecr.aws/l0g8r8j6
```
Comment (vincentni) [inline generatebundlefile/data/input_120_prod.yaml:24]: `public.ecr.aws/eks-anywhere`?

File: generatebundlefile/data/input_121_prod.yaml [hunk 4]
```diff
@@ -16,6 +16,14 @@ packages:
         registry: public.ecr.aws/eks-anywhere
         versions:
             - name: 0.21.1-971c31238aeea5f1bda6ec95caaf24993b304157
+  - org: cert-manager
+    projects:
+      - name: cert-manager
+        workloadonly: true
+        repository: cert-manager/cert-manager
+        registry: public.ecr.aws/l0g8r8j6
```
Comment (vincentni) [inline generatebundlefile/data/input_121_prod.yaml:24]: `public.ecr.aws/eks-anywhere`?

**Review by vincentni — COMMENTED** [review]
questions on some registries

**Review by jonahjon — CHANGES_REQUESTED** [review]
Let's do 2 PR's here.... one to the release.yaml file and then another to the Prod ones.

**Comment by jonahjon** [comment]
Also like vincent pointed out we need to change the prod bundle source to point to `public.ecr.aws/eks-anywhere`

**Comment by a-cool-train** [comment]
Breaking down the PRs.

**Pull request closed without merging**